app/gui/src/main/scala/org/bitcoins/gui/TaskRunner.scala

```diff
@@ -11,7 +11,7 @@ import scalafx.scene.control.{Alert, Label}
   *
   * Copied from [[https://github.com/scalafx/ScalaFX-Tutorials/blob/master/slick-table/src/main/scala/org/scalafx/slick_table/TaskRunner.scala]]
   */
-class TaskRunner(mainView: Node, glassPane: Node, statusLabel: Label) {
+class TaskRunner(mainView: Node, glassPane: Node) {
```

Comment:
You can't make this diff without also binding all `statusLabel.text` values to `GlobalData.statusText`